src/lib.rs

```diff
@@ -155,13 +147,13 @@ macro_rules! assert_ne {
                 if *left_val == *right_val {
                     panic!("assertion failed: `(left != right)`: {}\
                           \n\
-                          \nleft:  `{:?}`\
-                          \nright: `{:?}`\
+                          \n{}:\
+                          \n{:#?}\
                           \n\
                           \n",
                            format_args!($($arg)+),
-                           left_val,
-                           right_val)
+                           $crate::Style::new().bold().paint("Both sides"),
```

Comment:
I wonder how to style the titles (`Both sides` and `Diff`). Wonder if underlining wouldn't provide more visibility?

What do you think?